web/app/ratings_helpers.py

```diff
@@ -0,0 +1,22 @@
+
+
+def rating_to_traditional_range(rating_f):
```

Comment:
(i'm pretty happy about the fact that, with ZERO priors, zero anchors, zero thumbs-on-the-scale, we end up with a really close match on a couple dozen different players i looked up.  Would love to turn that into a quantifiable thing, but i'm really happy so far)